fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/rollcall/RollCallFragment.java

```diff
@@ -206,7 +207,24 @@ private void setUpStateDependantContent() {
     boolean isOrganizer = laoViewModel.isOrganizer();
 
     binding.rollCallFragmentTitle.setText(rollCall.getName());
-    binding.rollCallManagementButton.setVisibility(isOrganizer ? View.VISIBLE : View.GONE);
+    try {
+      // If the roll call is not closed management button is open or close
+      // Then it should be visible only to the organizer
+      // However, if isOrganizer but the roll call is closed then the button is
+      // reopen, however this is a valid action only if this is the last closed roll call
+      if (isOrganizer
```

Comment:
It would be good to extract some of the checks from the if() here as it gets a bit unclear what is actually checked.

Reply:
Yes, you're right, I'll try to simplify the readability